### PR TITLE
Update openms-ci.yml

### PR DESCRIPTION
**Pull request by timosachsenberg**
## Description

update qt in action

## Checklist
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes (Tick if no updates were necessary.)

### How can I get additional information on failed tests during CI
<details>
  <summary>Click to expand</summary>
If your PR is failing you can check out

- The details of the action statuses at the end of the PR or the "Checks" tab.
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. Use the "Show filters" capability on the top right to search for your PR number.
  If you click in the column that lists the failed tests you will get detailed error messages.

</details>

### Advanced commands (admins / reviewer only)
<details>
  <summary>Click to expand</summary>
  
- `/reformat` (experimental) applies the clang-format style changes as additional commit. Note: your branch must have a different name (e.g., yourrepo:feature/XYZ) than the receiving branch (e.g., OpenMS:develop). Otherwise, reformat fails to push.
- setting the label "NoJenkins" will skip tests for this PR on jenkins (saves resources e.g., on edits that do not affect tests)
- commenting with `rebuild jenkins` will retrigger Jenkins-based CI builds
  
</details>

---
:warning: Note: Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

**Review by jpfeuffer — COMMENTED** [review]
Try using
aqtversion: '==3.1.*'

and why not bumping to latest while you are at it?

**Comment by timosachsenberg** [comment]
Default: ==3.1.*

**Comment by jpfeuffer** [comment]
On master. We are at v3

File: .github/workflows/openms-ci.yml [hunk 1]
```diff
@@ -178,12 +179,13 @@ jobs:
...
       uses: jurplel/install-qt-action@v3
       with:
-        version: '5.12.7'
+        version: '5.15.4'
         host: 'mac'
         target: 'desktop'
         install-deps: 'true'
         cache: 'false'
         setup-python: 'false' # see https://github.com/jurplel/install-qt-action/issues/130
+        aqtversion: '==3.1.*'
```
Comment (timosachsenberg) [inline .github/workflows/openms-ci.yml:188]: probably not needed for mac
```suggestion
```

File: .github/workflows/openms-ci.yml
Comment (jpfeuffer) [inline .github/workflows/openms-ci.yml:1]: Remove that hardcoded line

Comment (timosachsenberg) [reply .github/workflows/openms-ci.yml:1]: which one? how to use latest qt 5?

File: .github/workflows/openms-ci.yml [hunk 2]
```diff
@@ -178,7 +179,7 @@ jobs:
     - name: Install Qt
       uses: jurplel/install-qt-action@v3
       with:
-        version: '5.12.7'
+        version: '5.15.2'
         host: 'mac'
         target: 'desktop'
         install-deps: 'true'
```
Comment (timosachsenberg) [inline .github/workflows/openms-ci.yml:185]:
```suggestion
        install-deps: 'true'
        aqtversion: '==3.1.*'
```

File: .github/workflows/openms-ci.yml
Comment (timosachsenberg) [reply .github/workflows/openms-ci.yml:1]: my concern is that they move it to 6.x in the future if I get rid of that line...

File: .github/workflows/openms-ci.yml [hunk 3]
```diff
@@ -213,17 +214,18 @@ jobs:
...
             do
               lnknm=$(basename "${f%/*}" ".framework")
               ln -s $f $lnknm
             done
-            for f in $Qt5_Dir/lib/*.framework/Versions/5/Qt*
+            for f in $Qt5_DIR/lib/*.framework/Versions/5/Qt*
             do
               install_name_tool -id $f $f
             done
-            otool -L /Users/runner/work/OpenMS/Qt/5.12.7/clang_64/lib/QtCore.framework/Versions/5/QtCore
```
Comment (jpfeuffer) [inline .github/workflows/openms-ci.yml:226]: this one. done.

**Comment by timosachsenberg** [comment]
version does not exist

**Review by jpfeuffer — APPROVED** [review]

**Pull request merged**